docs/getting-started/register-account.md

```diff
@@ -20,12 +20,12 @@ Please click on the "Verify Email" button that we sent to you via email to activ
 
 ![](./verify-account.png)
 
-Don't see this email in your inbox? Please check your spam folder and contact support@simplelogin.zendesk.com if the problem persists.
+Don't see this email in your inbox? Please check your spam folder and contact [support@simplelogin.zendesk.com](mailto:support@simplelogin.zendesk.com) if the problem persists
 
 ## Sign up with Proton
 
 If you have a Proton account, you can quickly sign up in SimpleLogin using the "Sign up with Proton" button. A new SimpleLogin will be created using your Proton address.
 
 If you have an Unlimited, Business and Visionary plan in Proton, your SimpleLogin account will be automatically upgraded.
 
-![](./sign-up-with-proton.png)
```

Comment:
what is the change here please? Github doesn't show the diff in a visible way.

Reply:
Where did you find a change? I am not author but here what I can see:

![IMG_1532](https://github.com/simple-login/documentation/assets/153930594/46d29f1d-0d1b-44bf-8cc4-21cc7110ef30)

Nothing changed. I think it is a glitch